buildkite/bazelci.py

```diff
@@ -1325,6 +1325,8 @@ def PrepareRepoInCwd(print_cmd_groups, initial_setup=False):
 
         execute_bazel_run(bazel_binary, platform, task_config.get("run_targets", None))
 
+        execute_bazel_mobile_install(bazel_binary, platform, task_config.get("mobile_install_targets", None))
```

Comment:
Do you really want to run mobile install targets before the normal build and test targets?